generators/templates/package.json

```diff
@@ -65,5 +65,8 @@
     "wct-browser-legacy": "^1.0.1",
     "web-component-tester": "^6.7.1"
   },<% } %>
-  "generator-pfelement-version": "<%= generatorPfelementVersion %>"
+  "generator-pfelement-version": "<%= generatorPfelementVersion %>",
+  "bugs": {
```

Comment:
Let's wrap this in `<% if (isPfelement) { %>` since we don't want bugs on non-pfelement components raised against the repo.

Reply:
Updated. Just curious, can you provide an example of a non-pfelement scenario? I thought the generator was only used to create elements.

Reply:
I can't but @kylebuch8 probably could.  He added the ability to create a vanilla web component that doesn't extend pfelement using this generator.